changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="Defra" id="1">
+      <tagDatabase tag="v1.1.0"/>
+    </changeSet>
+
+    <changeSet author="Defra" id="2">
+      <sql>
+        CREATE TYPE status_enum AS ENUM ('created', 'sending', 'delivered', 'permanent-failure', 'temporary-failure', 'technical-failure');
+      </sql>
+    </changeSet>
+
+    <changeSet author="Defra" id="3">
+    <dropTable  cascadeConstraints="true"
+            tableName="initial"/>
+
+    <createTable tableName="notifyApiRequestSuccess">
+      <column name="correlationId" type="UUID">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="notifyApiRequestSuccess_pkey"/>
+      </column>
+      <column name="createdAt" type="TIMESTAMP">
+        <constraints nullable="false"/>
+      </column>
+      <column name="notifyResponseId" type="VARCHAR(255)">
```

Comment:
Just realised this should be a UUID

Reply:
Ah thank you. Have updated the changelog.